content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -174,6 +174,41 @@ spec:
 
 However, the particular path specified in the custom recycler Pod template in the `volumes` part is replaced with the particular path of the volume that is being recycled.
 
+### Pre-binding
+
+The control plane can [bind Persistent Volume Claims to matching Persistent Volumes](#binding) in the
```

Comment:
@kbhawkey  we use PVC + PV throughout this doc. Do you think it's still better to spell out the abbreviation when we already introduced it earlier?

Reply:
Hi. Sorry for the confusion. I reread the paragraphs. When the terms are spelled out, it helps me understand the
concepts, especially when there is a number of cross referencing of fields in the configuration files. I am not against abbreviating the terms. The abbreviations are defined at the top of the document
and used throughout the page.

Is "Pre-binding" the correct heading or would "Preferred binding" or "Selective binding" make more sense?

Reply:
How about something like "Reserve a PersistentVolume for a PersistentVolumeClaim"?

In terms of abbreviation vs not, I'm fine either way if you think it's easier to understand. What do you think of keeping PersistentVolume and PersistentVolumeClaim as one word to match the object name?

Reply:
OK. The new section is found under the parent section, "Lifecycle of a volume and claim".
The other sub-section headings are one word.
One idea: Reserving a PersistentVolume?

Reply:
I like "Reserving a PersistentVolume"

Reply:
Updated.